tools/ktx/CMakeLists.txt

```diff
@@ -16,7 +16,9 @@ add_executable(ktxtools
         command_transcode.cpp
         command_validate.cpp
         deflate_utils.h
+        encode_utils_astc.h
```

Comment:
How did it build without this before?

Reply:
It builds because .cpp file(s)in this source list include this file. The only reasons to put .h files in this list are so they play a role in dependency checks and so they appear in the source file list in IDE projects. The latter is why I added it.